src/parser/jobs/war/index.tsx

```diff
@@ -17,7 +17,7 @@ export const WARRIOR = new Meta({
 
 	supportedPatches: {
 		from: '7.0',
-		to: '7.01',
+		to: '7.05',
```

Comment:
Can you please separate this patch bump into a separate PR?  Would rather not hold up WAR 7.05 support over this statistic.

Reply:
Makes sense - here is a seperate PR: https://github.com/xivanalysis/xivanalysis/pull/2080